src/main/java/sg/ncl/MainController.java

```diff
@@ -1608,15 +1631,18 @@ public String experiments(Model model, HttpSession session) {
                 for (int k = 0; k < experimentsArray.length(); k++) {
                     Experiment2 experiment2 = extractExperiment(experimentsArray.getJSONObject(k).toString());
                     Realization realization = invokeAndExtractRealization(experiment2.getTeamName(), experiment2.getId());
+                    System.out.println(realization.getDetails());
                     realizationMap.put(experiment2.getId(), realization);
                     experimentList.add(experiment2);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)